src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);

```

Comment:
You need to work with spark max because the motor is neo.
https://www.revrobotics.com/content/sw/max/sw-docs/java/index.html